dali/python/bundle-wheel.sh

```diff
@@ -52,13 +52,14 @@ OUTWHLNAME=${OUTWHLNAME//-none-/-}
 
 PKGNAME=$(echo "$OUTWHLNAME" | sed 's/-.*$//')
 PKGNAME_PATH=$(echo "$PKGNAME" | sed 's/_/\//' )
+# Strip cuda version from the name
+PKGNAME_PATH=${PKGNAME_PATH%%_cu*}
```

Comment:
does this mean that the final wheel doesn't have the cuda version in its package name?

Reply:
It has but we use that PKGNAME_PATH to generate directory structure when it installs, and I think we should keep with nvidia/dali. Without that change it makes something like nvidia/dali_cu100 for patched libs.